src/pages/auth/verify-email/form-verify-email.tsx

```diff
@@ -0,0 +1,134 @@
+import {useEffect, useMemo, useState} from "react";
+import {Controller, useForm} from "react-hook-form";
+import toast from "react-hot-toast";
+import {redirect, useNavigate, useParams} from "react-router-dom";
+import {zodResolver} from "@hookform/resolvers/zod";
+import useCountdown from "@hooks/use-countdown.ts";
+import {Box, CircularProgress, FormHelperText} from "@mui/material";
+import Button from "@mui/material/Button";
+import {authResendOTP, authVerifyEmail} from "@services/api/auth-api.ts";
+import {numberToTime} from "@utils/libs.ts";
+import {VerifyEmailSchema, VerifyEmailType} from "@utils/schema.ts";
+import {isAxiosError} from "axios";
+import {MuiOtpInput} from "mui-one-time-password-input";
+
+const FormVerifyEmail = () => {
+  const navigate = useNavigate();
+  const {email} = useParams();
+  const {current, reset} = useCountdown(0, 60 * 5);
```

Comment:
define constant for this code like 
```suggestion
  const {current, reset} = useCountdown(0, TIME_END);
```